fixtures/provider.py

```diff
@@ -301,12 +306,15 @@ def _get_template(provider, template_type_name):
             # Same as couple of lines above
             return template
         templates = TEMPLATES.get(provider.key)
-        if templates and template_name in templates:
+        # If template is type str then return template else template_name
+        # since template could be AttrDict and returning it would fail provisioning
+        # _get_template should always return template name of type string
+        if templates and template_name in templates and isinstance(template, str):
```

Comment:
`... and isinstance(template, six.string_types):`

We use `six.string_types` instead of `str` in these cases to make sure that things will continue working when we switch to Python 3.